contracts/generate-bindings.sh

```diff
@@ -23,9 +24,6 @@ function create_binding_abi_only {
   abigen --abi=data/tmp.abi --pkg=contract${contract} --out=$binding_dir/${contract}/binding.go
```

Comment:
should we add a ticket or potentially consider migrating to abigen v2 at some point? 
- https://github.com/ethereum/go-ethereum/pull/31379 
- http://geth.ethereum.org/docs/developers/dapp-developer/native-bindings-v2

Reply:
yes please do so. would be a nice to have but I think it will be non-trivial change to apply so not really a priority for me atm.

Reply:
when doing so we'll be able to more easily log all eth-call arguments as well, which will be a nice debugging benefit